src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -44,8 +48,6 @@ public static void assertSamsEqual(final File sam1, final File sam2, final Valid
 
     /**
      * causes an exception if the given sam files aren't equal
-     * @param reference is allowed to be null
-     * the default ValidationStringency value for this method is DEFAULT_STRINGENCY
      */
```

Comment:
Why did you delete this still-valid javadoc? As punishment, you should not only restore it, but add full javadoc for all parameters for public methods in this class.